README.md

```diff
@@ -8,11 +8,13 @@
 
 
 
-The wcf repo contains the library implementation for [Windows Communication Foundation](https://github.com/dotnet/wcf). It includes System.ServiceModel.Http, System.ServiceModel.NetTcp and other libraries. It builds and runs on Windows. You can ['watch'](https://github.com/dotnet/wcf/subscription) the repo to see progress as additional libraries are added.
+This repository contains the [.NET Core](http://github.com/dotnet/core) version of the [Windows Communication Foundation](https://msdn.microsoft.com/en-us/library/dd456779.aspx) client libraries.
 
-The version of WCF in this repo runs on top of the [.NET Core](http://github.com/dotnet/core) platform, namely the [.NET Core Runtime (CoreCLR)](https://github.com/dotnet/coreclr) and [.NET Core Framework (CoreFx)](http://github.com/dotnet/corefx)
+It's a subset of the .NET Framework version of Windows Communication Foundation and currently supports the same functionality available for Windows 8.1 Store apps. It is used to build .NET Core apps, including [Windows UWP](https://msdn.microsoft.com/en-us/library/windows/apps/dn609832.aspx) and [ASP.NET 5](http://weblogs.asp.net/scottgu/introducing-asp-net-5). These client libraries are suitable for mobile devices or on mid-tier servers to communicate with existing WCF services.
```

Comment:
Consider "currently supports the same functionality..." -> "will support the same functionality" or "currently implements the same contract... " ? 

I'm at a loss for words right now, but I want to say that we currently support the same *contract* as Windows Store 8.1 apps but the *functionality* is still work in progress.